src/factories/series/Area.ts

```diff
@@ -1,39 +1,69 @@
 module n3Charts.Factory.Series {
   'use strict';
 
-  export class Area extends Line {
-    containerClassName = 'area-data';
-    itemsClassName = 'area-series';
-    itemClassName = 'area';
+  export class Area extends Utils.SeriesFactory {
+
+    static type: string = Utils.Options.SERIES_TYPES.AREA;
```

Comment:
This should work in the other direction : each series should have its own type as a static property, and `Utils.Options` should read from them.